x-pack/plugin/searchable-snapshots/src/main/java/org/elasticsearch/xpack/searchablesnapshots/allocation/decider/HasFrozenCacheAllocationDecider.java

```diff
@@ -31,21 +31,21 @@ public class HasFrozenCacheAllocationDecider extends AllocationDecider {
     private static final Decision HAS_FROZEN_CACHE = Decision.single(
         Decision.Type.YES,
         NAME,
-        "this node has a frozen searchable snapshot shard cache"
+        "this node has a searchable snapshot shared cache"
     );
 
     private static final Decision NO_FROZEN_CACHE = Decision.single(
         Decision.Type.NO,
         NAME,
         "node setting ["
             + SHARED_CACHE_SIZE_SETTING.getKey()
-            + "] is set to zero, so frozen searchable snapshot shards cannot be allocated to this node"
+            + "] is set to zero, so shards of partially mounted indices cannot be allocated to this node"
     );
 
     private static final Decision UNKNOWN_FROZEN_CACHE = Decision.single(
         Decision.Type.NO,
         NAME,
-        "there was an error fetching the frozen cache state from this node"
+        "there was an error fetching the searchable snapshot shared cache state from this node"
```

Comment:
This is fine, though I think this really is the "partially mounted index shared cache". But this aligns with settings and documentation and I am mostly splitting hairs here.

Reply:
I also hesitated on using the "partially mounted index" terms and finally opted for using "searchable snapshot" that are more generic and hopefully easier to find in docs.